etc/policy.template.toml

```diff
@@ -0,0 +1,338 @@
+[loader]
+
+# How loaded zones are reviewed.
+#
+# Review offers an opportunity to perform external checks on the zone contents
+# loaded by Cascade.
+[loader.review]
+
+# Whether review is required.
+#
+# If this is 'true', a loaded version of a zone will not be signed or published
+# until the user explicitly approves it.  If it is 'false', loaded zones will be
+# signed immediately.  The review hook (below) will be run in any case.
+required = false
+
+# A hook for reviewing a loaded zone.
+#
+# This command string will be executed in the user's shell when a new version of
+# a zone is loaded.  At the moment, it will only be run if 'required' is true.
+#
+# It will receive the following information via environment variables:
+#
+# - 'CASCADE_ZONE': The name of the zone, formatted without a trailing dot.
+# - 'CASCADE_SERIAL': The serial number of the zone (decimal integer).
+# - 'CASCADE_TOKEN': A UUID associated with this review, for approving or
+#   rejecting this version of the zone.
+# - 'CASCADE_SERVER': The TCP/UDP port where Cascade is serving the zone for
+#   review, formatted '<ip-addr>:<port>'.
+# - 'CASCADE_CONTROL': The address of Cascade's HTTP API server, for sending
+#   approvals and rejections.
+#
+# To approve the zone contents, send an HTTP POST request to
+# `http://$CASCADE_SERVER/rs/approve/$CASCADE_TOKEN?zone=$CASCADE_ZONE&serial=$CASCADE_SERIAL`
+# (where `approve` can be replaced by `reject` to reject the zone).
+#
+# TODO: Use the exit code of the hook to infer approval / rejection?
+#cmd-hook = "review-unsigned-zone.sh"
+
+
+# DNSSEC key management.
+[key-manager]
+
+# The HSM server to use.
+#
+# If this is set, the named HSM server (which must be configured via 'cascade
+# hsm add') will be used for storing the zone's DNSSEC keys.
+#
+# TODO: Link to a guide for using HSMs with Cascade.
+# TODO: Allow using multiple HSMs, and HSMs in tandem with on-disk keys.
+#hsm-server-id = "softhsm"
+
+# How long keys are considered valid for.
+#
+# If a key has been used for longer than this time, it is considered expired,
+# and (if enabled) it will automatically be rolled over to a new key.  Even
+# if automatic rollovers are not enabled, the key will be reported as expired.
+# This is a soft condition -- DNSSEC does not have a concept of key expiry,
+# and it will not break DNSSEC validation, but it is usually important to the
+# security of the zone.
+#
+# Independent validity times are set for KSKs, ZSKs, and CSKs.  An integer
+# value will be interpreted as seconds; 'forever' means keys never expire.
+ksk.validity = "31536000" # TODO: "1Y"
+zsk.validity = "2592000" # TODO: "1M"
+csk.validity = "31536000"
+
+# Whether to automatically start key rollovers.
+#
+# If this is enabled, Cascade will automatically start rolling over keys when
+# they expire (as per 'validity').  When using this setting, 'validity' must not
+# be set to 'forever'.
+#
+# TODO: Maybe we want to start rolling over keys some time before expiry?
```

Comment:
Isn't that what validity is for?

Reply:
I suggest to leave the TODO out as I would want @Philip-NLnetLabs to comment here and I think it's better not to leave an uncertain TODO item in the policy template.